modules/configuring-high-availability/proc-configuring-high-availability-in-rhdh-operator-deployment.adoc

```diff
@@ -0,0 +1,25 @@
+[id="proc-configuring-high-availability-in-rhdh-operator-deployment"]
+= Configuring High Availability in a {product} Operator deployment
+
+When you are deploying {product-short} using the Operator, you must set `replicas` to a value greater than `1` in your `{product-custom-resource-type}` CR.
+
+
+.Procedure
+
+. In your `{product-custom-resource-type}` CR, set `replicas` to a value greater than `1`. For example:
```

Comment:
For single-step procedures, use an unnumbered bullet instead of a numbered list. See [redhat-documentation.github.io/modular-docs#con-creating-procedure-modules_writing-mod-docs](https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs)


```suggestion
* In your `{product-custom-resource-type}` custom resource, set `replicas` to a value greater than `1`. For example:
```